tests/libinfra/network.go

```diff
@@ -1,34 +0,0 @@
-/*
- * This file is part of the KubeVirt project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * Copyright 2023 Red Hat, Inc.
- *
- */
-
-package libinfra
-
-import (
-	"github.com/onsi/gomega"
-	v1 "k8s.io/api/core/v1"
-
-	"kubevirt.io/kubevirt/tests/libnet"
-)
-
-func GetSupportedIP(ips []string, family v1.IPFamily) string {
-	ip := libnet.GetIP(ips, family)
-	gomega.ExpectWithOffset(1, ip).NotTo(gomega.BeEmpty())
```

Comment:
The `libnet` version does not have this assertion.
It may be fine, but this is an observed difference.

Reply:
The callers of the method that may run in ipv6 only cluster have `libnet.SkipWhenClusterNotSupportIPFamily(family)`.